<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditVideoWindow</class>
 <widget class="QMainWindow" name="EditVideoWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>435</width>
    <height>234</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Úprava videa</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Název:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit">
            <property name="readOnly">
             <bool>false</bool>
            </property>
            <property name="placeholderText">
             <string>Název videa</string>
            </property>
            <property name="clearButtonEnabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <spacer name="verticalSpacer_5">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>22</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox">
            <item>
             <property name="text">
              <string>.mp4</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>.mp3</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>.wav</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>.ogg</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_4">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Začátek:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTimeEdit" name="timeEdit">
            <property name="currentSection">
             <enum>QDateTimeEdit::SecondSection</enum>
            </property>
            <property name="displayFormat">
             <string>HH:mm:ss</string>
            </property>
            <property name="currentSectionIndex">
             <number>2</number>
            </property>
            <property name="time">
             <time>
              <hour>0</hour>
              <minute>0</minute>
              <second>0</second>
             </time>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>15</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Konec:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTimeEdit" name="timeEdit_2">
            <property name="currentSection">
             <enum>QDateTimeEdit::SecondSection</enum>
            </property>
            <property name="displayFormat">
             <string>HH:mm:ss</string>
            </property>
            <property name="currentSectionIndex">
             <number>2</number>
            </property>
            <property name="time">
             <time>
              <hour>0</hour>
              <minute>0</minute>
              <second>0</second>
             </time>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>15</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QPushButton" name="pushButton_2">
            <property name="text">
             <string>Reset hodnot</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="text">
             <string>Odejít</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_3">
            <property name="text">
             <string>Potvrdit</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>435</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMo_nosti">
    <property name="title">
     <string>Možnosti</string>
    </property>
    <addaction name="actionVymazat_p_vodn_soubor"/>
   </widget>
   <addaction name="menuMo_nosti"/>
  </widget>
  <action name="actionVymazat_p_vodn_soubor">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Vymazat původní soubor</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
